NuGet.config

```diff
@@ -77,6 +77,7 @@
       <package pattern="commandlineparser" />
       <package pattern="diffplex" />
       <package pattern="fluentassertions" />
+      <package pattern="gee.external.capstone" />
```

Comment:
I assume this is just a dependency of the new versions?

Reply:
Yes, it's a dependency of the newer version of BDN. It's a disassembly library.